airgun/views/errata.py

```diff
@@ -40,7 +40,7 @@ def search(self, query, applicable=True, installable=False, repo=None):
         if repo is not None:
             self.repo_filter.fill(repo)
 
-        if re.search(r'\w{3,4}[:-]\d{4}[-:]\d{4}', query):
+        if re.search(r'\w{3,4}[:-]\d{4}[-:]\d{1,4}', query):
```

Comment:
@sambible  @vsedmik @vijaysawant  Do you see any value for this regex? As per I know we never use the Debian errata, mostly test RHEL errata.

Reply:
We don't test any Debian errata that I'm aware of. This will change this particular regex to allow for 
DSA-1234-1 where in the past it would not have matched, since RHEL errata are in the form
RHSA-1234-1234 ( With explicitly 4 characters in the last third) 

These will still get captured by this adjusted regex, although it won't match them as strictly.  If there's value to @dosas here, I'd be curious to know the use case

Reply:
My use case is to read and install debian errata with robottelo which currently is not possible since the [search](https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/errata.py#L138) is a requirement for the ErrataDetailsView

Reply:
Upon writing the above comment I realized that the same regexp is also used for this [view ](https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/errata.py#L140).

So if there is consens on accepting this PR I would maybe put this information in one place?